File: library/src/main/java/com/snail/easyble/core/GattDescriptor.java
package com.snail.easyble.core;

import android.support.annotation.NonNull;

import java.util.UUID;

/**
 * 描述:
 * 时间: 2018/7/24 10:01
 * 作者: zengfansheng
 */
public class GattDescriptor {
    public UUID serviceUuid;
    public UUID characteristicUuid;
    public UUID descriptorUuid;
    public byte[] value;

    public GattDescriptor(@NonNull UUID serviceUuid, @NonNull UUID characteristicUuid, @NonNull UUID descriptorUuid, byte[] value) {
        this.serviceUuid = serviceUuid;
        this.characteristicUuid = characteristicUuid;
        this.descriptorUuid = descriptorUuid;
        this.value = value;
    }
}

File: library/src/main/java/com/snail/easyble/core/GattDescriptor.kt
package com.snail.easyble.core

import java.util.*

/**
 * 描述:
 * 时间: 2018/7/24 10:01
 * 作者: zengfansheng
 */
class GattDescriptor(var serviceUuid: UUID, var characteristicUuid: UUID, var descriptorUuid: UUID, var value: ByteArray)
